db/seeds/items.rb

```diff
@@ -1,10 +1,10 @@
 category = Category.find_by_name("T-shirts")
-category.items.create(name: "Mens  T-shirt", description: "100% Cotton", image_url: "http://flathash.com/fake_url_01")
-category.items.create(name: "Womens T-shirt", description: "100% Cotton", image_url: "http://flathash.com/fake_url_02")
+category.items.create(name: "Mens  T-shirt", description: "100% Cotton", image_url: "http://flathash.com/fake_url_01", price: 10.43)
+category.items.create(name: "Womens T-shirt", description: "100% Cotton", image_url: "http://flathash.com/fake_url_02", price: 11.43)
 
 category = Category.find_by_name("HouseHold")
-category.items.create(name: "Mug", description: "100% Ceramic", image_url: "http://flathash.com/fake_url_03")
+category.items.create(name: "Mug", description: "100% Ceramic", image_url: "http://flathash.com/fake_url_03", price:4.43 )
```

Comment:
Line is too long. [122/80]<br>Space missing after colon.<br>Space inside parentheses detected.